media/j2store/css/J4/j2store_admin.css

```
@@ -1,5 +1,5 @@
-body.com_j2store .page-title {background:url(../images/j2store-icon.png) no-repeat 0 50%;padding-left:50px;}
-@media (max-width: 767px) { body.com_j2store .page-title {background:url(../images/j2store-icon.png) no-repeat center center;padding-left:0; text-indent: -800px} }
+body.com_j2store .page-title {background:url(../../images/j2store-icon.png) no-repeat 0 50%;padding-left: 25px;margin-left: 25px;}
```

```suggestion
body.com_j2store .page-title {background:url(../../images/j2store-icon.png) no-repeat 0 50%;padding-inline-start: 25px;margin-inline-start: 25px;}
```

logical css operators ensire its both LTR and RTL compatible